### DISCUSSION TITLE
Intellichlor Low Salt Indication

### DISCUSSION THREAD

**Discussion by mzuniga51** [tagyoureit/nodejs-poolController#426] [General]
@rstrouse would it be is possible to gather the Low Salt alarm data directly from my Intellichlors and into grafana?

**Comment by rstrouse** [comment]
Yes that is emitted as status.  It will include all of the status reported from the chlorinators.  Why are you not just reporting the salt level?

**Comment by mzuniga51** [comment]
I'm trying to understand at what salt level will the cell trigger a low salt alert.  I have realized my two cells trigger it at different salt levels for some reason and my salt reading from the cell seem to be quite higher than the Taylor kit-measured salt which I believe is uncommon .   Some in TFP say that just add salt whenever the cell ask for it and not to worry about the actual reading two much but I'm not too convinced on that.  Anyways,  I don't see any status reported from the chlorinators.  See what I get in the picture below.
<img width="518" alt="Screen Shot 2022-04-24 at 21 07 34" src="https://user-images.githubusercontent.com/91519307/165014664-9aad8ea8-be9a-4985-9c93-df5620b67883.png">

**Reply by rstrouse** [reply]
Neither test is terribly accurate.  Both of them measure the salts in the water.  When I say salts I mean all salts in the water.  This is known as total dissolved solids.  Then the scale is adjusted for the TDS factor of fresh water using a constant.  I believe the uses an electrical conductivity sensor and the Taylor kit simply measures the ph after the silver nitrate reacts with the sals leaving nitric acid.

**Comment by mzuniga51** [comment]
Right, when a test has an accuracy of +-400 ppm on 3000 ppm well, you know.   Now, either way, it would be cool to be able to get a notice once the cell thinks there is low salt, whatever that means, because I imagine this will be the point where the cell will stop producing chlorine altogether.  Any way to see that?

**Comment by rstrouse** [comment]
I believe it is emitted as a tag since it has no scale.  It should be in your database.

EDIT: You should not be getting low salt warning unless it is below 2800 and very low salt at less than 2600.  It does not shut down until very low salt.
